web/search.dart

```diff
@@ -233,7 +273,12 @@ void initializeSearch(
         event.preventDefault();
       }
     });
-
+    var suggestionEnclose = match.enclosedBy;
+    if (suggestionEnclose != null) {
+      var category = '${suggestionEnclose.name} ${suggestionEnclose.type}';
+      var toMap = createCategory(category, suggestionEnclose.href);
```

Comment:
Hmm, I don't understand this variable name. But to simplify things, you could just inline the `createCategory` call, haha:

```dart
categorize(createCategory(...`
```